client/service/src/builder.rs

```diff
@@ -528,7 +528,7 @@ where
 			keystore.clone(),
 			system_rpc_tx.clone(),
 			&config,
-			backend.offchain_storage(),
+			backend.clone(),
```

Comment:
so previously we're only accessing offchain storage and now we need full storage, right?